src/react/features/filters/hooks/useTextFilter.js

```diff
@@ -0,0 +1,66 @@
+import { useEffect } from 'react'
+import { useDispatch, useStore } from '../../../store'
+import { useCompare } from '../../../utils/useCompare'
+
+import { setFocus } from '../../navigation/actions'
+import { addDiory, updateDiory } from '../../room/actions'
+
+const SEARCH_TOOL_ID = 'SEARCH_TOOL_ID'
```

Comment:
This is defined in two places (also in useTurnOnFilters).

I don't know what is the correct place but this is incorrect! :D